tests/automation/utilities/set_disappearing_messages.ts

```diff
@@ -50,25 +54,24 @@ export const setDisappearingMessages = async (
     // Check that 1 Day default is automatically selected (default time is only relevant to 1:1's)
     let defaultTime;
     if (timerType === 'disappear-after-read-option') {
-      defaultTime = await waitForElement(
-        windowA,
-        'data-testid',
-        'input-time-option-43200-seconds',
-      );
+      // making explicit DataTestId here as `waitForElement` currently allows a string
```

Comment:
We should probably update that to use matching selector to Strategy type, can you add a TODO here?

Reply:
added todo twice